frontend/packages/ceph-storage-plugin/src/components/odf-system/actions.tsx

```diff
@@ -0,0 +1,20 @@
+import { K8sKind } from '@console/dynamic-plugin-sdk';
+import { K8sResourceKind, referenceForModel } from '@console/internal/module/k8s';
+import { Kebab } from '@console/internal/components/utils';
+import { addCapacityModal } from '../modals/add-capacity-modal/add-capacity-modal';
+import { OCSServiceModel } from '../../models';
+
+const addStorage = (kind: K8sKind, resource: K8sResourceKind) => ({
+  labelKey: 'ceph-storage-plguin~Add Capacity',
+  callback: () => addCapacityModal({ kind, ocsConfig: resource }),
+});
```

Comment:
Can you check you are not getting any i18n errors ?
Otherwise do something like:
```suggestion
const addStorage = (kind: K8sKind, resource: K8sResourceKind) => ({
  // t('ceph-storage-plguin~Add Capacity',)
  labelKey:t('ceph-storage-plguin~Add Capacity',),
  callback: () => addCapacityModal({ kind, ocsConfig: resource }),
});
```